package.xml

```diff
@@ -16,15 +16,15 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>boost</build_depend>
-  <build_depend>libconsole-bridge-dev</build_depend>
+  <build_depend>console_bridge_vendor</build_depend>
   <build_depend>urdfdom_headers</build_depend>
   <build_depend>urdfdom_py</build_depend>
   <build_depend>tinyxml2_vendor</build_depend>
   <build_depend>urdf</build_depend>
 
   <exec_depend>urdf</exec_depend>
   <exec_depend>boost</exec_depend>
-  <exec_depend>libconsole-bridge-dev</exec_depend>
```

Comment:
I think you can depend on both `libconsole-bridge-dev` **and** `console_bridge_vendor`.
This way `rosdep` will install the upstream version of `console_bridge` in Debian, and if (at some point in the future) it meets the minimum version requirement of `console_bridge_vendor`, we can avoid building it from source.